packages/components/pie-tag/test/component/pie-tag.spec.ts

```diff
@@ -17,4 +46,68 @@ test.describe('PieTag - Component tests', () => {
         // Assert
         expect(tag).toBeVisible();
     });
+
+    test.describe('icon slot', () => {
+        test.describe('when passed', () => {
```

Comment:
You could remove one of these describe blocks and just do the follow:

```javascript
test.describe('when passed in with a size `large`', () => {
```

Reply:
I updated the test to include "when NOT passed" scenario as well so now it makes more sense as is